website/templates/website/error_pages/404.html

```diff
@@ -1,9 +1,19 @@
+{% load staticfiles %}
 {% extends "website/base_template.html" %}
 
-{% block content_body %}
+<link href='https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@600;900&display=swap' rel='stylesheet' type='text/css'>
+{% block extra_css %}{% endblock %}
+<link rel="stylesheet" type="text/css" href="{% static "css/styles404.css" %}">
+
+ 
+<script src="https://kit.fontawesome.com/4b9ba14b0f.js" crossorigin="anonymous"></script>
+{% block extra_js_top %}{% endblock %}
 
-<!-- Content -->
-<div class="content-card">
-  <div class="page-heading"><h2>404 Not Found</h2></div>
-</div>
+{% block content_body %}
+    <div class="mainbox">
+      <div class="err">4</div>
+      <i class="far fa-question-circle fa-spin"></i>
+      <div class="err2">4</div>
+      <div class="msg">Maybe this page moved? Got deleted? Is hiding out in quarantine? Never existed in the first place?<p>Let's go <a href="index.html">home</a> and try from there.</p></div>
```

Comment:
this link does not work and does not redirect to home folder `<a href="index.html">`

Reply:
@skoudoro  can you help me with this ? what should I give the link as?

Reply:
you need to use django syntax: `<a href="{% url 'index' %}">`

Reply:
Yeah! @skoudoro  the URL problem is resolved.